src/routes/formsgSiteLaunch.ts

```diff
@@ -63,24 +63,13 @@ export class FormsgSiteLaunchRouter {
       },
     ]
 
-    // todo remove this after local dev is done
-    const isDev = true
-
-    if (isDev || redirectionDomain) {
+    if (redirectionDomain) {
       subDomainSettings.push({
         branchName: "master",
         prefix: "www",
```

Comment:
FYI - I have some concerns about the redirection logic which was covered in #546.

Reply:
Ok, lets address it directly there then!